README.md

```diff
@@ -1 +1,163 @@
-# ros-frontend
\ No newline at end of file
+# Resource Optimization Service (Frontend)
+
+Resource Optimization is a service that can help you optimize your public cloud workloads on Red Hat Enterprise Linux (RHEL). Read more about it [here](https://www.redhat.com/en/blog/optimize-public-cloud-workloads-rhel-red-hat-insights-resource-optimization)
+
+## Prerequisites
+
+### Install
+
+- [Node](https://nodejs.org/en/download/)
+- [Docker](https://docs.docker.com/get-docker/)
+
+
+## Setup Development Environment
+
+1. Clone below repositories:
+
```

Comment:
Since these are valid commands, we may show them in monospace code?

Reply:
@gnurag make sense, updated!